catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -331,4 +334,11 @@ private void closeNotificationBar() {
 			sendBroadcast(it);
 		}
 	}
+
+	public static int getUniqueNotificationId() {
+		if (uniqueNotificationIdCounter == null) {
+			uniqueNotificationIdCounter = new AtomicInteger(0);
+		}
```

Comment:
https://github.com/Catrobat/Catroid/pull/2379#discussion_r128917046 includes to remove static from getUniqueNotificationId as well
Instead of calling `StatusBarNotificationManager.getUniqueNotificationId()` you use
`StatusBarNotificationManager.getInstance().getUniqueNotificationId()`
Let me know if you have further questions.